data/json/dreams.json

```diff
@@ -489,6 +505,14 @@
             ],
         "category" : "RAPTOR",
         "strength" : 3
+    },{
+        "type" : "dream",
+        "messages" : [
+            "You weave between the zombies' legs, too small for them to catch.  The gas station isn't far now, with all its soda and chips and- UGH, dream...",
+            "Once you finish your tasty dinner of seeds, you look around for your drink, but you already ate it."
```

Comment:
You ate the drink?

Reply:
I believe a [reference to acquiring all their hydration needs through eating](https://en.wikipedia.org/wiki/Kangaroo_rat#cite_ref-Lidicker1960_9-0).

Reply:
It's referencing metabolic rehydration, yeah. Rewording could be a thing.

Reply:
Ah alright. I thought it referenced your lack of impulse control or something.